crates/miden-testing/src/kernel_tests/tx/test_account_delta.rs

```diff
@@ -182,7 +182,7 @@ async fn storage_delta_for_value_slots() -> anyhow::Result<()> {
         .await
         .context("failed to execute transaction")?;
 
-    let storage_values_delta = executed_tx
+    let storage_items_delta = executed_tx
```

Comment:
Changes in this fine should be reverted: it contains the account's storage, which shouldn't be changed.